src/tools/meson-flex-wrapper

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
```

Comment:
I'd not name this meson-flex-wrapper. I'd probably just go for pgflex or such? But it doesn't really matter...